src/classes/CallableApi.cls

```diff
@@ -47,19 +47,22 @@ global with sharing class CallableApi implements System.Callable {
     * action handler based on the action text
     * @param action string of the target action
     * @param args name value map of arguments
-    * @return Object returned will depend on the results of the target function
+    * @return Object returned will depend on the results of the target function otherwise it returns true
     ********************************************************************************************************/
     global Object call(String action, Map<String, Object> params) {
        
         switch on (action.toLowerCase()) {
             when 'tdtm.disablealltriggers' {
-                return TDTM_Config_API.disableAllTriggers();
+                TDTM_Config_API.disableAllTriggers();
```

Comment:
@force2b 
If action = 'TDTM.DisableAllTriggers' then action.toLowerCase() should convert it to 'tdtm.disablealltriggers' .  Unless I'm missing something?

a null action would throw an error but we are expecting an action

Reply:
That can work to. I was just trying to separate the two parts of the action name (UPPERCASE.lowercase). Doesn't really matter though. This meets the need perfectly.

Do you mean a null response would throw an exception?

Reply:
yeah because action.toLowerCase() will throw an error if action is null right?  Should we check for that before doing the toLowerCase()?

Reply:
Ah. The inline reassignment example I had from earlier could catch that, or we could just let the null value throw the npe exception. I like the idea of us throwing our custom exception ("(null) is not implemented", or something along those lines).

Reply:
Added a check for the text and formatted the exception message to put in the ( parentheses )

Reply:
Thanks @patrick-yan-sf. The parenthesis was only for the null value. It would be better as just wrapping action in single quotes, so a null or empty string would thrown an exception of `'' is not implemented`.